server/routes/tool_permissions.py

```
@@ -0,0 +1,122 @@
+_ERR_NO_ORG = "No org context"
+
+
+def _invalidate_permissions_cache(org_id: str) -> None:
+    """Set Redis dirty flag so running chats refresh permissions on next tool call."""
+    try:
+        from utils.cache.redis_client import get_redis_client
+        rc = get_redis_client()
+        if rc:
+            rc.set(f"tool_perms_dirty:{org_id}", "1", ex=3600)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)